main/settings.py

```diff
@@ -7,6 +7,14 @@
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
 
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = 'DEBUG' in os.environ
+
+# SECURITY WARNING: keep the secret key used in production secret!
+if DEBUG or True:
```

Comment:
Oops I was using this for debugging when I wasn't using dockerfile. I'll remove it